docs/sidebars.ts

```diff
@@ -3,83 +3,94 @@ import type {SidebarsConfig} from "@docusaurus/plugin-content-docs";
 const sidebars: SidebarsConfig = {
   tutorialSidebar: [
     "index",
-    "introduction",
-    {
-      type: "doc",
-      label: "Security",
-      id: "security",
-    },
```

Comment:
I can't find deep references to those page anywhere, imo removing content should be separate from a "layout change" PR